instructions-for-devs.md

```diff
@@ -0,0 +1,174 @@
+# Instructions for Developers
+
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. 
+
+Developers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend cloning the [latest release](https://github.com/ExchangeUnion/xud/releases):
+```bash
+git clone https://github.com/ExchangeUnion/xud -b 1.0.0-prealpha
+cd xud
+npm install
+```
+
+
+Xud uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You will have to install one of those and create a user *xud* and a database *xud* and grant all permissions for the new database to the *xud* user.
+
+## 2. Prepare your environment 
+
+To initialize the database with some testing data, run the following command from `~/xud`:
+
+```bash
+npm run db:init
+```
+
+Setup raiden according to [this guide](https://github.com/ExchangeUnion/xud/blob/raiden_swap/lib/raidenclient/README.md).
+
+[Releases](https://github.com/ExchangeUnion/xud/releases) currently only support raiden ERC20 swaps on testnet, thus we recommend to disable lnd in the config, to avoid unnecessary error outputs:
+```toml
+[lnd]
+disable = true
+```
+
+## 3. Starting the Daemon
+
+Open a new terminal and launch a `xud` process
+
+```bash
+~/xud/bin $ ./xud
+2018-6-3 15:14:27 [GLOBAL] info: config loaded
+2018-6-3 15:14:27 [DB] info: connected to database. host:localhost port:3306 database:xud
+2018-6-3 15:14:28 [P2P] info: pool server listening on 0.0.0.0:8885
+2018-6-3 15:14:28 [RPC] info: GRPC server listening on port 8886
+2018-6-3 15:14:28 [RPC] info: gRPC Web API proxy listening on port 8080
+```
+
+Optional command line arguments to override defaults and settings in the [configuration](#configuration) file for a specific `xud` instance
```

Comment:
Should be `#5-configuration-optional` instead of `#configuration`

Reply:
Good catch!